tox.ini

```diff
@@ -28,6 +27,15 @@ commands_post =
     # This is to deal with running the containers with --detached
     docker-compose down
 
+[testenv:docs]
+commands_pre =
+    pip install poetry
+    poetry install
+    # Generate the requirements file for Readthedocs
+    poetry export -f requirements.txt -o docs/requirement.txt
```

Comment:
@ahal is there a better way for readthedocs to not require generating the requirements file by exporting and then checking in the file manually?